### ISSUE TITLE
guava does not get installed (no CHANGES.guava)

### ISSUE THREAD

**Issue by dimpase** [cschwan/sage-on-gentoo#770]
```
>>> Source compiled.
>>> Test phase [not enabled]: dev-gap/guava-3.17

>>> Install dev-gap/guava-3.17 into /var/tmp/portage/dev-gap/guava-3.17/image
Traceback (most recent call last):
  File "/usr/lib/portage/python3.11/doins.py", line 609, in <module>
    sys.exit(main(sys.argv[1:]))
             ^^^^^^^^^^^^^^^^^^
  File "/usr/lib/portage/python3.11/doins.py", line 598, in main
    if _doins(
       ^^^^^^^
  File "/usr/lib/portage/python3.11/doins.py", line 441, in _doins
    return install_runner.install_file(source, os.path.dirname(dest))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/portage/python3.11/doins.py", line 375, in install_file
    return self._ins_runner.run(source, dest_dir)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/portage/python3.11/doins.py", line 183, in run
    sstat = os.stat(source)
            ^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: b'CHANGES.guava'
 * ERROR: dev-gap/guava-3.17::sage-on-gentoo failed (install phase):
 *   dodoc failed
```

One can probably also use guava 3.18 instead of 3.17.

**Comment by kiwifb** [comment]
I failed QA there, a change in the files from the previous version. Thanks for reporting. The fix and 3.18 will get merged with the rest of sage 10.2 later this morning.

**Comment by kiwifb** [comment]
Done.
